File: tickers/src/main/java/br/com/jeffersonbraga/tickers/TickersApplication.java
package br.com.jeffersonbraga.tickers;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

@SpringBootApplication
@EnableDiscoveryClient
public class TickersApplication {

	public static void main(String[] args) {
		SpringApplication.run(TickersApplication.class, args);
	}

}

File: tickers/src/main/java/br/com/jeffersonbraga/tickers/TickersApplication.kt
package br.com.jeffersonbraga.tickers

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder
import org.springframework.boot.context.ApplicationPidFileWriter
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

@SpringBootApplication
@EnableDiscoveryClient
open class TickersApplication

fun main(args: Array<String>) {

    val app = SpringApplicationBuilder(TickersApplication::class.java)
    app.build().addListeners(ApplicationPidFileWriter("./bin/shutdown.pid"))
    app.run(*args)
}
